src/ucs/sys/event_set.c

```diff
@@ -20,7 +20,10 @@
 #  include "config.h"
 #endif
 
-#define UCS_EVENT_EPOLL_MAX_EVENTS 16
+
+const unsigned ucs_sys_event_set_max_events =
+    ucs_static_floor(UCS_ALLOCA_MAX_SIZE / sizeof(struct epoll_event));
```

Comment:
what is the difference between this and a regular integer division?

Reply:
yes, this is enough to just do `UCS_ALLOCA_MAX_SIZE / sizeof(struct epoll_event)`, it will be rounded up correctly

Reply:
i guess you meant rounded _down_ ..

Reply:
> i guess you meant rounded _down_ ..

yes, that's what I meant